pyproject.toml

```diff
@@ -1,6 +1,9 @@
 [tool.poetry]
 name = "uw-webdriver-recorder"
-version = "4.0.0"
+# This version string is typically managed by the CI workflow,
+# and is changed anytime `poetry version [new version]` is run.
+# Do not revert this manually.
+version = "4.0.1-alpha.0"
```

Comment:
I generated a prerelease using the manual flow for verifying the fix with another package.

Reply:
(NB: This has since been updated to `4.0.1-alpha.1`)